add HttpClient configration with Student configration

MohanadYousif opened the pull request:
Please one of you look at my httpclient code and approve it  if everything looks good. Thanks

Commit:
add HttpClient configration with Student configration

ViceSyndicate commented:
I noticed line 22 in StudentServices.cs returns null but that's not a big deal for me. 
I also noticed a Student.cs class in the Models folder. I'm not sure what purpose it serves, could you tell me?

Commit:
Update IService.cs

Change method name from in IService.cs interface from GetAsync to GetByIdAsync.

Commit:
Update StudentService.cs

Update StudentService.cs class to implement IService interface
